torch/_higher_order_ops/wrap.py

```diff
@@ -47,7 +47,7 @@ class TagActivationCheckpoint(HigherOrderOperator):
     """
 
     def __init__(self):
-        super().__init__("wrap_activation_checkpoint")
+        super().__init__("tag_activation_checkpoint")
```

Comment:
what is this? and why is 'wrap_activation_checkpoint' still left in the case statement in torchvariable?

Reply:
oh i see WrapActivationCheckpoint still exists.  Is WrapActivationCheckpoint doing things differently, not by passing information to aot about what to partition?

Reply:
This is just something I missed in the last PR. We have both wrap and tag activation_checkpoint higher order ops right now. This is just a temporary stage. 

wrap_activation_checkpoint - This asks AOT Autograd to trace torch.utils.checkpoint code.
tag_activation_checkpoint - This just tags the nodes to be recomputable, and then partitioner actually duplicates the nodes. We avoid tracing through the torch.utils.checkpoint. This is currently preferred because here we use inductor functionalization for rng states.

Reply:
This points to the other HigherOrderOp - https://github.com/pytorch/pytorch/pull/103953/files#diff-bc653d54b1dbf34241875a12617c6a2a86be63bce7543cca65bd38d7ed2d5afeL24

Reply:
got it. please also add that explanation of the 2 behaviors as a code comment, sounds useful.  (you almost have it, you have the half about what tagAC does but not as clear about the wrapAC one

Reply:
Done!